accounts/keystore/account_cache_test.go

```diff
@@ -106,8 +110,14 @@ func TestWatchNoDir(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 
 	// Create the directory and copy a key file into it.
-	os.MkdirAll(dir, 0700)
-	defer os.RemoveAll(dir)
+	if err := os.MkdirAll(dir, 0700); err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err := os.RemoveAll(dir); err != nil {
+			panic(err)
```

Comment:
These two panic calls make the test significantly harder to read for no benefit. Please remove.